modules/cda/routes/cda.ts

```diff
@@ -412,4 +411,18 @@ router.get('/:id/:name', async (req: any, res, next) => {
 });
 
 
+router.delete('/:idCda', async (req: any, res, next) => {
+    try {
+        const idCda = ObjectId(req.params.idCda);
+        const cdaFile = makeFs();
+        const file = await cdaFile.findOne({ _id: idCda });
+        if (file && file._id) {
+            await cdaFile.unlink(file._id, (error) => { });
```

Comment:
La función [unlink](https://github.com/lykmapipo/mongoose-gridfs/blob/master/DOCUMENTATION.md#fileschemastaticsunlink_id-done) no devuelve una promesa.  Usa el estandar de callbacks lamentablemente. 

```suggestion
            cdaFile.unlink(file._id, (error) => {
                if (error)  { return next(error); }
                return res.json({ success: true });
            });
```

Como está el código actualmente siempre va a devolver _success_ aunque haya algún error.